Resources/Prototypes/ai_factions.yml

```diff
@@ -83,3 +84,55 @@
   - Zombie
   - SimpleHostile
   - Dragon
+
+- type: npcFaction
+  id: Nature
+  hostile:
+  - Zombie
+  - VoxSpecies
+  - ArachnidSpecies
+  - DwarfSpecies
+  - GingerbreadSpecies
+  - HumanSpecies
+  - MothSpecies
+  - ReptilianSpecies
+  - SkeletonSpecies
+  - SlimeSpecies
+  friendly:
+  - Dragon
+  - PetsNT
+  - Xeno
+  - Mouse
+  - DionaSpecies
+
+#Species factions
+
+- type: npcFaction
+  id: ArachnidSpecies
+  
+- type: npcFaction
+  id: DionaSpecies
+  
+- type: npcFaction
+  id: DwarfSpecies
+  
+- type: npcFaction
+  id: GingerbreadSpecies
+  
+- type: npcFaction
+  id: HumanSpecies
+  
+- type: npcFaction
+  id: MothSpecies
+  
+- type: npcFaction
+  id: ReptilianSpecies
+  
+- type: npcFaction
+  id: SkeletonSpecies
+  
+- type: npcFaction
+  id: SlimeSpecies
+  
+- type: npcFaction
+  id: VoxSpecies
```

Comment:
unless you plan on doing something with all of these, there's not really a need to add all of these individually.

Reply:
I haven't found a better way to make monsters aggressive to everyone but one species.
This could also be used for other race-dependent mechanics in the future.

Reply:
This can be used to make spiders neutral to the arachnid race, all slimes neutral to the slime race

Reply:
> I haven't found a better way to make monsters aggressive to everyone but one species. This could also be used for other race-dependent mechanics in the future.

There's faction exceptions.